app/main.py

```diff
@@ -1,25 +1,83 @@
 from abc import ABC
+from typing import Callable, Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = name
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: object, owner: type) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: object, value: Any) -> None:
+        if not isinstance(value, int):
+            raise TypeError
+
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
```

Comment:
```suggestion
        if not isinstance(value, int):
            raise TypeError("Some informative message....")

        if not (self.min_amount <= value <= self.max_amount):
            raise ValueError("Some informative message....")
```
It is good practice to add an informative message to the error.
Please do this.

Reply:
fixed that, but readme.md says "Error messages can be skipped"